src/auth/session/storage/mysql.ts

```diff
@@ -82,6 +82,34 @@ export class MySQLSessionStorage implements SessionStorage {
     return true;
   }
 
+  public async deleteSessions(ids: string[]): Promise<boolean> {
+    await this.ready;
+    const query = `
+      DELETE FROM ${this.options.sessionTableName}
+      WHERE id IN (${ids.map(() => '?').join(',')});
+    `;
+    await this.query(query, ids);
+    return true;
+  }
+
+  public async findSessionsByShop(
+    shop: string,
+  ): Promise<SessionInterface[] | {[key: string]: unknown}[] | undefined> {
+    await this.ready;
+    const query = `
+      SELECT * FROM ${this.options.sessionTableName}
+      WHERE shop = ?;
+    `;
+    const [rows] = await this.query(query, [shop]);
+    if (!Array.isArray(rows) || rows?.length === 0) return undefined;
+
+    const results: SessionInterface[] = [];
+    for (const row of rows) {
```

Comment:
I think we can simplify this a bit by using `map` instead of a loop. I'm assuming we can probably do that for all SQL-based classes?